### ISSUE TITLE
VGGNet on CIFAR-10

### ISSUE THREAD

**Issue by lgzz** [he-y/filter-pruning-geometric-median#37]
![image](https://user-images.githubusercontent.com/22911283/66905329-3c2e1a80-f038-11e9-9627-6b721498c9ed.png)
I'm trying to reproduce this results, but I got worse results, the accuracy of Pruned without FT I got are much lower than the results in this table, which is about 25%(PFEC). Could you upload your pre-trained checkpoint that is used to prune?

**Comment by he-y** [comment]
The models are already released at README, please find them on [this link](https://drive.google.com/drive/u/1/folders/1hGnULraEbz8IjSRZx_juzZnvDTDqDdt-).

**Comment by lgzz** [comment]
I have looked for that carefully, but there is no pre-trained model. I really doubt this result, so I want to get your model.

**Comment by he-y** [comment]
The pre-trained models are clearly located at VGG/baseline, why can't you find them?
![image](https://user-images.githubusercontent.com/10541284/66917199-84a50280-f04f-11e9-9796-f52b8eed8cba.png)


Please take a look at the snapshot from the log file:
**Prune wo finetune:**
![image](https://user-images.githubusercontent.com/10541284/66916887-cda88700-f04e-11e9-915d-cbf6a4db2fe2.png)

**40 epoch finetune:**
![image](https://user-images.githubusercontent.com/10541284/66917401-e4031280-f04f-11e9-93d6-76b7fdb1e920.png)

**160 epoch finetune:**
![image](https://user-images.githubusercontent.com/10541284/66917438-f67d4c00-f04f-11e9-9275-114a30179f77.png)

This is where our results come from.

**Comment by lgzz** [comment]
There is no baseline folder under VGG. Maybe you forgot to share it.

**Comment by he-y** [comment]
I carefully checked my files and found one of the directories are not shared. Maybe I drag this directory in after setting the parent folder as "share mode", so this folder wasn't shared.
![image](https://user-images.githubusercontent.com/10541284/66918409-e2d2e500-f051-11e9-9867-24f79a6a56d9.png)

**Comment by he-y** [comment]
Sorry about that. 
Now I share the folder. Please take a look and check it.

**Comment by lgzz** [comment]
OK. Thank you very much.
